assets/dev/scss/editor/preview/preview.scss

```diff
@@ -23,6 +23,43 @@
 
 @import "../flex-icons.scss";
 
+
+// mixins
+@mixin e-grid-outline-base {
+	padding-top: var( --padding-top );
+	padding-bottom: var( --padding-bottom );
+	display: grid;
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	pointer-events: none;
+	//grid-template-columns: var(--e-con-grid-template-columns);
+	//grid-template-rows: var(--e-con-grid-template-rows);
+	grid-gap: var( --gap);
+	justify-items: var(--justify-items);
+	align-items: var(--align-items);
+	grid-auto-flow: var(--grid-auto-flow);
+	justify-content: var(--grid-justify-content);
+	align-content: var(--grid-align-content);
+	min-height: var(--min-height);
+}
+
+@mixin e-grid-outline-boxed {
+	@include e-grid-outline-base;
+	//max-width: calc( var(--content-width ) - ( (var( --padding-right ) + var( --padding-left ) ) / 2 ) * 1.6 );
+	//max-width: var(--content-width);
```

Comment:
@davseve Can we remove the commented out code in this entire file?

Reply:
Done,
Sorry for that